query_execution/QueryExecutionMessages.proto

```diff
@@ -0,0 +1,33 @@
+//   Copyright 2015-2016 Pivotal Software, Inc.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+syntax = "proto2";
+
+package quickstep.serialization;
+
+// Used for both Normal WorkOrders and RebuildWorkOrders.
+message WorkOrderCompletionMessage {
```

Comment:
I agree that for now, we can live with single message type for both of them but separating messages for WorkOrder completion and Rebuild WorkOrder completion is better. 
By design, RebuildWorkOrders are always going to be about rebuilding a single block. Where as a WorkOrder itself could mean various things with respect to the input - e.g. hash tables, multiple blocks (e.g. sort). If we need to make any choices in the scheduler about the recently executed workorder and its input and output data, we will need to provide the information about the data locations through the completion messages. That's where separating the two message types will come handy.